packages/frontend/components/Borrow/PreviewTransaction.tsx

```diff
@@ -0,0 +1,311 @@
+import React, { useState } from "react"
+import { useTheme } from "@mui/material/styles"
+import {
+  Box,
+  ButtonBase,
+  Card,
+  CardActionArea,
+  CardContent,
+  Collapse,
+  Container,
+  Dialog,
+  DialogContent,
+  Divider,
+  Grid,
+  Tooltip,
+  Typography,
+} from "@mui/material"
+import InfoOutlinedIcon from "@mui/icons-material/InfoOutlined"
+import AddCircleIcon from "@mui/icons-material/AddCircle"
+import CancelIcon from "@mui/icons-material/Cancel"
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore"
+import Image from "next/image"
+
+import LTVProgressBar from "./LTVProgressBar"
+import ClickableTooltip from "../Layout/ClickableTooltip"
+
+export default function PreviewTransaction() {
+  const { palette } = useTheme()
+  const [showProvider, setShowProvider] = useState(false)
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null)
+  const isOpen = Boolean(anchorEl)
+
+  const openMenu = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget)
+  }
+
+  const closeMenu = () => {
+    setAnchorEl(null)
+  }
+  console.log(isOpen)
+
+  return (
+    <Container
```

Comment:
same remark as above I think you can simply remove it and put padding on `Grid` above